drivers/hmis/app/graphql/types/hmis_schema/household.rb

```diff
@@ -12,19 +12,16 @@ class HmisSchema::Household < Types::BaseObject
     field :id, ID, null: false
     field :short_id, ID, null: false
     field :household_clients, [HmisSchema::HouseholdClient], null: false
+    field :household_size, Int, null: false
 
-    # object is a scope on Hmis::Hud::Enrollment
-
-    def id
-      object.first.household_id
-    end
+    # object is a Hmis::Hud::Household
 
     def short_id
-      object.first.short_household_id.upcase
+      object.enrollments.first.short_household_id.upcase
```

Comment:
This seems unnecessary to pull up enrollments, when we alreayd have the id.
Can we move/copy [these](https://github.com/greenriver/hmis-warehouse/blob/5ec51d47d9a961bac93b9b034092bf3e451a0b5b/drivers/hmis/app/models/hmis/hud/enrollment.rb#L180-L183) to the new household model as a `def short_id`? Then we don't need this function on the type anymore.